recipes-samples/packagegroups/packagegroup-rpb-tests.bb

```diff
@@ -45,6 +46,7 @@ RDEPENDS:packagegroup-rpb-tests-console = "\
     libgpiod-tools \
     lmbench \
     ltp \
+    make \
```

Comment:
rpb-console-image-test should already have packagegrou-core-buildessential, which should bring make. so that should not be needed.

Reply:
@ndechesne : indeed 'make' is there. I think the test team was not correct with their observations, but I found the same available already:

root@sa8155p-adp:~# make
make: *** No targets specified and no makefile found.  Stop.

root@sa8155p-adp:~# which make
/usr/bin/make

root@sa8155p-adp:~# cmake
-sh: cmake: command not found

I will send an updated PR shortly.

Thanks.